config/yugabyte/regression_pipelines/orderby_workloads/yugabyte/ORDG7_orderby_on_hash_partitioned_table.yaml

```diff
@@ -115,19 +123,51 @@ microbenchmark:
                   - name: ordby_bigint_secondary_indexed_column
                     weight: 100
                     queries:
-                        - query: select col_bigint_id_4 from Indexed1M_1 order by col_bigint_id_4
+                        - query: select col_bigint_id_3 from Indexed1M_1 order by col_bigint_id_3
             - workload: ORDG7_ordby_bigint_range_secondary_indexed_with_filter_column
               customTags: customer=1,schematype=regular,partition=hash,skey=asc,cardinality=single,projection=indexed,filtercount=one,filteron=rangeskey,filtertype=comparision,queryshape=orderby,orderbyon=rangeskey
               run:
                   - name: yb_ordby_bigint_secondary_indexed_column
                     weight: 100
                     queries:
-                        - query: select col_bigint_id_4 from Indexed1M_1 where col_bigint_id_4>1 order by col_bigint_id_4
+                        - query: select col_bigint_id_3 from Indexed1M_1 where col_bigint_id_3>1 order by col_bigint_id_3
 
             - workload: ORDG7_ordby_bigint_hash_pkey_with_filter_column_desc
-              customTags: customer=1,schematype=regular,partition=hash,pkey=hash,cardinality=single,projection=hashpkey,filtercount=one,filteron=hashpkey,filtertype=comparision,queryshape=orderby,orderbyon=hashpkeydesc
+              customTags: customer=1,schematype=regular,partition=hash,pkey=composite,cardinality=multiple,projection=hashpkey,filtercount=one,filteron=hashpkey,filtertype=comparision,queryshape=orderby,orderbyon=hashpkeydesc
+              run:
+                  - name: yb_ordby_bigint_non_indexed_column
```

Comment:
Query names are same for newly added queries, please modify them to have unique names.

Reply:
I have changed the query name. Though I am not sure where this name gets reflected. I know we perform comparison based on workload name. It would be good to know.